corehq/apps/app_manager/views/download.py

```diff
@@ -234,9 +234,16 @@ def resolve_path(path):
                 payload = request.app.fetch_attachment(full_path)
             except ResourceNotFound:
                 if build_profile in request.app.build_profiles and build_profile_access:
-                    request.app.create_build_files(save=True, build_profile_id=build_profile)
-                    request.app.save()
-                    payload = request.app.fetch_attachment(full_path)
+                    try:
+                        # look for file guaranteed to exist if profile is created
+                        request.app.fetch_attachment('files/{id}/profile.xml')
```

Comment:
Is this a format string, or is `{id}` a special folder name? Looks like a format parameter was missed.